### DISCUSSION TITLE
AMD 公開超強運算力電腦處理器，採台積電 5 奈米製程打造

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#631] [General]
https://buzzorange.com/techorange/2022/05/24/computex-amd/

超微 AMD 董事長暨執行長蘇姿丰於 23 日在 COMPUTEX CEO 主題演講中，正式發表下一代「Ryzen 7000」桌上型處理器，是首款採用台積電 5 奈米製程打造的桌機晶片，採用全新 Zen 4 架構，預計今年秋季推出。

**Comment by JSHT** [comment]
除了核心使用 TSMC 5nm，在 L2 快取也翻倍，加上新架構、改為 LGA 腳位能提供的高功率，能夠上到 DDR5-6000 CL30、全核 5.5 GHz。同時也都加上 RDNA2 架構的內顯、PCIe Gen5，真的蠻猛的。
AMD 這次透過聯合多家廠商一起進行硬體開發，加速更新產品迭代也是大家喜聞樂見的。

**Comment by Ben6uo** [comment]
但真正沒測試過之前都不知道 看看intel的11代就知道

**Comment by jason79461385** [comment]
但奈米數的減少也代表cpu內積熱量的增加，這就要看AMD他們要如何解決的
